<?xml version="1.0" encoding="UTF-8"?>
<!-- Generated by org.testng.reporters.JUnitXMLReporter -->
<testsuite ignored="0" hostname="Toshif" failures="1" tests="10" name="Test" time="18.55" errors="0" timestamp="10 Oct 2023 16:15:38 GMT">
  <testcase classname="com.organization.OrganizationTest" name="organizationPage1" time="9.181"/>
  <testcase classname="com.PractiseExcersise.MyPractiseTest" name="dataProvider" time="0.0"/>
  <testcase classname="com.PractiseExcersise.MyPractiseTest" name="dataProvider" time="0.0"/>
  <testcase classname="com.PractiseExcersise.MyPractiseTest" name="dataProvider" time="0.0"/>
  <testcase classname="com.PractiseExcersise.MyPractiseTest" name="date1" time="0.0"/>
  <testcase classname="com.PractiseExcersise.MyPractiseTest" name="excelWrite" time="0.063">
    <failure type="java.lang.IllegalArgumentException" message="The workbook already contains a sheet named &amp;apos;Toshif&amp;apos;">
      <![CDATA[java.lang.IllegalArgumentException: The workbook already contains a sheet named 'Toshif'
at org.apache.poi.xssf.usermodel.XSSFWorkbook.validateSheetName(XSSFWorkbook.java:914)
at org.apache.poi.xssf.usermodel.XSSFWorkbook.createSheet(XSSFWorkbook.java:867)
at org.apache.poi.xssf.usermodel.XSSFWorkbook.createSheet(XSSFWorkbook.java:123)
at com.PractiseExcersise.MyPractiseTest.excelWrite(MyPractiseTest.java:65)
... Removed 29 stack frames]]>
    </failure>
  </testcase> <!-- excelWrite -->
  <testcase classname="com.PractiseExcersise.MyPractiseTest" name="excelread" time="0.203"/>
  <testcase classname="com.PractiseExcersise.MyPractiseTest" name="getSystemDateInFormat" time="0.0"/>
  <testcase classname="com.PractiseExcersise.MyPractiseTest" name="propertyfile" time="0.0"/>
  <testcase classname="com.PractiseExcersise.MyPractiseTest" name="random" time="0.0"/>
</testsuite> <!-- Test -->
